acme-sh-helper

```diff
@@ -28,34 +33,65 @@ while test -n "$1"; do
   esac
 done
 
-if [ "$DOMAIN" == "" ] ; then
+if [ ${#DOMAINS[@]} -eq 0 ] ; then
     echo "==> Please provide a --domain foo.tld argument."
     exit 1
 fi
 
-export NSUPDATE_SERVER="ns0.ukcod.org.uk"
-export NSUPDATE_KEY="/data/letsencrypt/etc/acme.key"
+if [ ${#DOMAINS[@]} -gt 1 ] && [ "$WILDCARD" ] ; then
+    echo "==> Please only specify one --domain foo.tld argument for a wildcard certificate."
+    exit 1
+fi
+
+
+if [ "$STAGING" ] ; then
+    SERVER="https://acme-staging-v02.api.letsencrypt.org/directory"
+else
+    SERVER="https://acme-v02.api.letsencrypt.org/directory"
+fi
+
+DOMAIN=${DOMAINS[0]}
```

Comment:
I think we'll need to cater for group certificates - see comment on `letsencrypt.py` changes.